diff -Naur dji_thermal_sdk_v1.3_20220517/sample/dji_ircm.cpp dji_thermal_sdk_v1.3_20220517_patched/sample/dji_ircm.cpp
--- dji_thermal_sdk_v1.3_20220517/sample/dji_ircm.cpp	2022-05-17 19:38:50.000000000 +1000
+++ dji_thermal_sdk_v1.3_20220517_patched/sample/dji_ircm.cpp	2022-06-12 00:09:54.000000000 +1000
@@ -29,6 +29,7 @@
 #include <iterator>
 #include <vector>
 #include <string.h>
+#include <math.h>
 #include <sys/stat.h>
 
 #include "dirp_api.h"
